.github/workflows/build.yml

```diff
@@ -0,0 +1,24 @@
+name: Build
+on: [push, pull_request]
+
+jobs:
+  Playnite-sln:
+    runs-on: windows-2019
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    - name: Setup Nuget.exe
+      uses: nuget/setup-nuget@v1
+    - name: Restore packages
+      run: nuget restore source/Playnite.sln
+
+    - name: Copy font files to avoid build errors
+      run: |
+        cp references\Fonts\TitilliumWeb-Black.ttf references\Fonts\PlayStation4.ttf
+        cp references\Fonts\TitilliumWeb-Black.ttf references\Fonts\XBOXONE.ttf
```

Comment:
This isn't an amazing workaround for the font issue. Ideally, there would be some authenticated file store that CI could access through [repository-level secrets](https://docs.github.com/en/actions/reference/encrypted-secrets).

Reply:
I believe one solution would be to set up a private repository `JosefNemec/fonts`, then have the build script checkout that repository and copy the font files.